pkg/validation/options.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/util"
 )
 
+const (
+	CodeChallengeMethodPlain = "plain"
+	CodeChallengeMethodS256  = "S256"
+)
```

Comment:
This should be in the API not the validation package

Reply:
Moved!